VK_App/VK_App/Controller/PageMyFriendViewController.swift

```diff
@@ -0,0 +1,76 @@
+//
+//  PageMyFriendViewController.swift
+//  VK_App
+//
+//  Created by Анастасия Козлова on 01.11.2022.
+//
+
+import UIKit
+
+/// Страница пользователя(друга)
+final class PageMyFriendViewController: UIViewController {
+    
+    // MARK: - Private Constant
+    private enum Constant {
+        static let pageOneIDText = "page1"
+        static let pageButtonIDText = "buttonCell"
+        static let cellTypes: [CellTypes] = [.name, .buttons ]
+    }
+    
+    private enum CellTypes {
+        case name
+        case buttons
+    }
+    
+    // MARK: - Life cycle
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+    }
+}
+
+// MARK: - Подписываемся на делегаты UICollectionViewDelegate, UICollectionViewDataSource, UICollectionViewDelegateFlowLayout
```

Comment:
```suggestion
// MARK: UICollectionViewDelegate, UICollectionViewDataSource
```
UICollectionViewDelegateFlowLayout этот делегат вместе с его функцией, которая ниже, вынеси в отдельное расширение этого контроллера , просто ниже расширения для collectionView